### PR TITLE
fix(design audit): updated styles.ts for multiple components, added d…

### PR DESCRIPTION
**Pull request by hthomas-czi**
…isabled style to defaultTheme

## Summary

**Design Audit**
Shortcut ticket: [sh-215620](https://app.shortcut.com/sci-design-system/story/215620/review-designs-and-code-for-fixes-that-need-to-be-made)

## Checklist

- [ ] Default Story in Storybook
- [ ] LivePreview Story in Storybook
- [ ] Test Story in Storybook
- [ ] Tests written
- [ ] Variables from `defaultTheme.ts` used wherever possible
- [ ] If updating an existing component, depreciate flag has been used where necessary
- [ ] Chromatic build verified by @chanzuckerberg/sds-design

### CODE REVIEW

**Review by clarsen-czi — CHANGES_REQUESTED** [review]
Looking good to me @hthomas-czi ! Just a few small tweaks needed that I noticed. Of course, I was just looking at the visual side of this, so definitely see what eng says from a code perspective.

File: src/core/CellHeader/style.ts [hunk 1]
```diff
@@ -76,8 +78,8 @@ export const StyledTableHeader = styled("th", {
       &:hover {
         color: ${active ? colors?.primary[500] : "black"};
 
-        & .MuiButtonBase-root {
-          color: ${active ? colors?.primary[500] : colors?.gray[500]};
+        & > .MuiButtonBase-root {
```
Comment (bethbertozzi) [inline src/core/CellHeader/style.ts:81]: What is the expected behavior here? I noticed when I remove the `>` a dropdown appears when I hover over the text, and when it is there the dropdown isn't there:
Screen Shot 2022-11-10 at 10 14 54 AM
Screen Shot 2022-11-10 at 10 15 06 AM

File: src/core/InputDropdown/style.ts [hunk 2]
```diff
@@ -161,22 +161,19 @@ const square = (props: InputDropdownProps): SerializedStyles => {
 
   return css`
     border-radius: ${corners?.m}px;
-    height: 34px;
```
Comment (bethbertozzi) [inline src/core/InputDropdown/style.ts:164]: was this doing anything before?

File: src/core/InputToggle/style.ts [hunk 3]
```diff
@@ -80,7 +82,7 @@ const toggleOn = (props: InputToggleExtraProps) => {
         color: ${disabled ? colors?.gray[300] : colors?.gray[600]};
         content: "${value}";
         font: inherit;
-        font-family: 'Open sans';
+        font-family: ${typography?.fontFamily};
```
Comment (bethbertozzi) [inline src/core/InputToggle/style.ts:85]: nice fix 🙌

File: src/core/Tabs/style.tsx [hunk 4]
```diff
@@ -75,16 +75,11 @@ export const StyledTab = styled(RawTab, {
...
-    const { sdsSize } = props;
-
-    const isLarge = sdsSize === "large";
-
     return `
       font-weight: ${fontWeights?.semibold};
       margin: 0 ${spaces?.xl}px ${spaces?.xxs}px 0;
 
       // (thuang): Large Tab height is 30px, the offset is 4px
-      height: ${isLarge ? 26 : 22}px;
```
Comment (bethbertozzi) [inline src/core/Tabs/style.tsx:87]: why was this deleted? Also, if it should be deleted the comment should be as well

**Review by bethbertozzi — COMMENTED** [review]
Great work! I have questions about a few of the changes but other wise the clean up looks great - thanks for this 🎉

File: src/core/CellHeader/style.ts (hunk 1 above)
Comment (hthomas-czi) [reply src/core/CellHeader/style.ts:81]: I'm not sure where this change came from. I don't see it in my commit history, but maybe I'm missing something. It's unintentional though. Do I need to make a new commit and rerun the tests to make this change?

File: src/core/InputDropdown/style.ts (hunk 2 above)
Comment (hthomas-czi) [reply src/core/InputDropdown/style.ts:164]: The height of this component should be implicitly calculated from the line height and padding rather than hard-coded.

File: src/core/Tabs/style.tsx (hunk 4 above)
Comment (hthomas-czi) [reply src/core/Tabs/style.tsx:87]: Same rational as the `InputDropdown`. The height should be implicitly calculated from the line height and padding rather than hard-coded. 

Thanks for pointing out the comment. I should have deleted that.

File: src/core/CellHeader/style.ts (hunk 1 above)
Comment (bethbertozzi) [reply src/core/CellHeader/style.ts:81]: hmm maybe it was by mistake? the '>' is not on the master branch so lets remove it.

**Review by bethbertozzi — APPROVED** [review]

**Comment by masoudmanson** [comment]
This PR was created before our transition to a monorepo package structure, necessitating a substantial number of changes (666, to be precise) for compatibility. Considering the extensive time and effort required to merge these changes, I've decided to create a new pull request, which encompasses all the improvements. You can find the updated changes in the new PR: #742. I'll be closing this PR now.

Thanks for all the improvements, @hthomas-czi! Apologies for the delay in merging this PR.
